src/Settings.tsx

```diff
@@ -225,6 +241,15 @@ function SettingsPopup({show, close}: SettingsPopupProps): JSX.Element {
             onValueChange={value => setChatModel(value)}>
             {['gpt-3.5-turbo', 'gpt-4', 'gpt-4-turbo-preview'].map(value => <Picker.Item label={value} value={value} key={value}/>)}
           </Picker>
+          <Text style={styles.text}>System Instructions</Text>
+          <FluentTextInput
+            accessibilityLabel="System Instructions"
+            multiline={true}
+            style={{flexGrow: 1, minHeight: 80}}
+            value={systemInstructions}
```

Comment:
```suggestion
            multiline={true}
            value={systemInstructions}
```
This styling should be handled by FluentTextInput. If it isn't working, fix that style rather than overriding.